app/src/server/router/occupies.ts

```diff
@@ -0,0 +1,20 @@
+import { createRouter } from "./context";
+import { z } from "zod";
+
+export const occupiesRouter = createRouter()
```

Comment:
See above with protected vs normal router? If so, why is this one not protected router and the other one is?

Reply:
This should be protected, good catch. Will fix this now